CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Version 1.4.6 
+- Fixed an issue affecting prechat forms, that would stop a visitor from starting a chat.
+- Fixed an issue that visitors couldn't start a chat, due to a side effect of keyboard registration events.
```

Comment:
- Fixed an issue that could stop some visitors from starting a chat, due to a side effect of keyboard registration events.